marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/AbstractSparkConnectorTest.java

```diff
@@ -157,4 +159,24 @@ protected GenericInternalRow buildRow(String... values) {
         }
         return new GenericInternalRow(rowValues);
     }
+
+    /**
+     * @param options
+     * @return
+     * */
+    protected DataWriter<InternalRow> buildStreamWriter(Options options) {
```

Comment:
We don't need this test, as it's not testing anything new. The DataWriter that's being returned is identical to the one being returned by buildDataWriter - it's being constructed in the same fashion, since OutputMode doesn't matter. . 

What we need tests for are the commit/abort methods that take an epoch and don't take an epoch. But, we don't have any logic for those yet. That's where we need some research to figure out when Glue/Spark is invoking these methods - we should add some logging and then test out the connector to see when these methods are invoked, and what's in the WriterCommitMessage objects that are passed to it.